boards/nxp/imx91_evk/imx91_evk_mimx9131_defconfig

```diff
@@ -28,3 +28,6 @@ CONFIG_CONSOLE=y
 CONFIG_UART_CONSOLE=y
 
 CONFIG_CLOCK_CONTROL=y
+
+# I2C
+CONFIG_I2C=y
```

Comment:
No need to enable I2C in board defconfig as you don't enable any i2c device

Reply:
Agree, will remove it. Thanks.